testing/conformance/src/vm.rs

```diff
@@ -107,7 +107,7 @@ impl TestMachine<Box<DefaultMachine<MemoryBlockstore, TestExterns>>> {
 
         let mut nc = NetworkConfig::new(network_version);
         nc.override_actors(builtin_actors);
-        let mut mc = nc.for_epoch(epoch, state_root);
+        let mut mc = nc.for_epoch(epoch, (epoch * 30) as u64, state_root);
```

Comment:
timestamp = 30 * chain epoch num?

Reply:
Yes. This is for testing only because we need a number. We'll probably need a better number later, but we can fix that later.